Tests/PowerSkillTests/Helpers.cs

```diff
@@ -96,9 +100,27 @@ public static HttpResponseMessage RespondRequestWith(this HttpRequestMessage req
         }
 
         public static Func<HttpRequest, Task<IActionResult>> CurrySkillFunction(Func<HttpRequest, ILogger, Microsoft.Azure.WebJobs.ExecutionContext, Task<IActionResult>> skillFunction)
-            => request => skillFunction(request, new LoggerFactory().CreateLogger("local"), new Microsoft.Azure.WebJobs.ExecutionContext());
+        {
+            var executionContext = new Microsoft.Azure.WebJobs.ExecutionContext();
+            var executingLocation = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
```

Comment:
We do this so much, we should just build a small helper for it. Not necessarily in this PR though.